PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -641,20 +682,20 @@ - (NSDictionary *)enumValuesOfType:(NSString *)type
                               @"sepia": @(PSPDFAppearanceModeSepia),
                               @"night": @(PSPDFAppearanceModeNight)},
 
-//                        @"PSPDFDocumentSharingOptions":
-//
-//                            @{@"None": @(PSPDFDocumentSharingOptionNone),
-//                              @"CurrentPageOnly": @(PSPDFDocumentSharingOptionCurrentPageOnly),
-//                              @"PageRange": @(PSPDFDocumentSharingOptionPageRange),
-//                              @"AllPages": @(PSPDFDocumentSharingOptionAllPages),
-//                              @"AnnotatedPages": @(PSPDFDocumentSharingOptionAnnotatedPages),
-//                              @"EmbedAnnotations": @(PSPDFDocumentSharingOptionEmbedAnnotations),
-//                              @"FlattenAnnotations": @(PSPDFDocumentSharingOptionFlattenAnnotations),
-//                              @"AnnotationsSummary": @(PSPDFDocumentSharingOptionAnnotationsSummary),
-//                              @"RemoveAnnotations": @(PSPDFDocumentSharingOptionRemoveAnnotations),
-//                              @"FlattenAnnotationsForPrint": @(PSPDFDocumentSharingOptionFlattenAnnotationsForPrint),
-//                              @"OriginalFile": @(PSPDFDocumentSharingOptionOriginalFile),
-//                              @"Image": @(PSPDFDocumentSharingOptionImage)},
+                        //                        @"PSPDFDocumentSharingOptions":
+                        //
+                        //                            @{@"None": @(PSPDFDocumentSharingOptionNone),
+                        //                              @"CurrentPageOnly": @(PSPDFDocumentSharingOptionCurrentPageOnly),
+                        //                              @"PageRange": @(PSPDFDocumentSharingOptionPageRange),
+                        //                              @"AllPages": @(PSPDFDocumentSharingOptionAllPages),
+                        //                              @"AnnotatedPages": @(PSPDFDocumentSharingOptionAnnotatedPages),
+                        //                              @"EmbedAnnotations": @(PSPDFDocumentSharingOptionEmbedAnnotations),
+                        //                              @"FlattenAnnotations": @(PSPDFDocumentSharingOptionFlattenAnnotations),
+                        //                              @"AnnotationsSummary": @(PSPDFDocumentSharingOptionAnnotationsSummary),
+                        //                              @"RemoveAnnotations": @(PSPDFDocumentSharingOptionRemoveAnnotations),
+                        //                              @"FlattenAnnotationsForPrint": @(PSPDFDocumentSharingOptionFlattenAnnotationsForPrint),
+                        //                              @"OriginalFile": @(PSPDFDocumentSharingOptionOriginalFile),
+                        //                              @"Image": @(PSPDFDocumentSharingOptionImage)},
```

Comment:
What is the purpose of this commented block? Can we remove it as a part of clean-up or if it is really needed for something can we add a comment on its purpose?

Reply:
The Sharing API has changed in v8 and we commented out the old sharing API. We created https://github.com/PSPDFKit/Cordova-iOS/issues/66 and we left it there commented out to re-use parts of it.